diff --git a/ImgurHolo/build.gradle b/ImgurHolo/build.gradle
index 8305ade..6511ea7 100644
--- a/ImgurHolo/build.gradle
+++ b/ImgurHolo/build.gradle
@@ -1,24 +1,18 @@
 buildscript {
     repositories {
         mavenCentral()
-        maven { url 'http://download.crashlytics.com/maven' }
 
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.8.+'
-        classpath 'com.crashlytics.tools.gradle:crashlytics-gradle:1.+'
-
+        classpath 'com.android.tools.build:gradle:0.8.1'
     }
 }
 apply plugin: 'android'
-apply plugin: 'crashlytics'
 
 repositories {
     mavenCentral()
     mavenLocal()
     mavenRepo urls:"http://maven.mashape.com/releases"
-    maven { url 'C:\\Users\\Ellen\\.m2\\repository' }
-    maven { url 'http://download.crashlytics.com/maven' }
 }
 
 
@@ -36,16 +30,16 @@ dependencies {
     compile 'com.google.code.gson:gson:2.2.4'
     compile files('libs/scribe-1.3.5.jar')
     compile files('libs/ion.jar')
+    compile project(':DragSort')
     compile files('libs/android-async.jar')
-    compile 'com.mobeta.android.dslv:drag-sort-listview:0.6.1-SNAPSHOT-AAR'
-    compile files('libs/android-support-v13.jar')
-    compile 'com.crashlytics.android:crashlytics:1.+'
+    compile 'com.android.support:support-v13:19.1.0'
 
 }
 
+
 android {
     compileSdkVersion 19
-    buildToolsVersion "19.0.1"
+    buildToolsVersion "19.0.3"
 
     defaultConfig {
         minSdkVersion 14
diff --git a/ImgurHolo/src/main/AndroidManifest.xml b/ImgurHolo/src/main/AndroidManifest.xml
index 441419b..061dbaf 100644
--- a/ImgurHolo/src/main/AndroidManifest.xml
+++ b/ImgurHolo/src/main/AndroidManifest.xml
@@ -14,7 +14,7 @@
         android:allowBackup="true"
         android:icon="@drawable/icon"
         android:label="imgur Holo"
-        android:debuggable="false"
+        
             >
         <activity
             android:name="com.krayzk9s.imgurholo.activities.SettingsActivity"
@@ -155,7 +155,6 @@
         </activity>
         <service android:name="com.krayzk9s.imgurholo.services.UploadService" />
         <service android:name="com.krayzk9s.imgurholo.services.DownloadService" />
-        <meta-data android:name="com.crashlytics.ApiKey" android:value="83ec4d6737e0b139ec006da8f1c6afe9b5eb36c4"/>
     </application>
 
 </manifest>
diff --git a/ImgurHolo/src/main/java/com/krayzk9s/imgurholo/activities/MainActivity.java b/ImgurHolo/src/main/java/com/krayzk9s/imgurholo/activities/MainActivity.java
index ebfe457..3c316ad 100644
--- a/ImgurHolo/src/main/java/com/krayzk9s/imgurholo/activities/MainActivity.java
+++ b/ImgurHolo/src/main/java/com/krayzk9s/imgurholo/activities/MainActivity.java
@@ -45,7 +45,6 @@ import android.widget.ListView;
 import android.widget.TextView;
 import android.widget.Toast;
 
-import com.crashlytics.android.Crashlytics;
 import com.krayzk9s.imgurholo.R;
 import com.krayzk9s.imgurholo.services.UploadService;
 import com.krayzk9s.imgurholo.tools.ApiCall;
@@ -72,7 +71,6 @@ public class MainActivity extends ImgurHoloActivity implements GetData {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        Crashlytics.start(this);
 
         setContentView(R.layout.activity_main);
         updateMenu();

diff --git a/build.gradle b/build.gradle
index f3d8cd8..8a4a2cc 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,4 +1,31 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-dependencies {
-    project(":ImgurHolo")
-}
\ No newline at end of file
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:0.8.1'
+    }
+}   
+
+
+project(':DragSort') {
+ 
+    apply plugin: 'android-library'
+
+    dependencies {
+      compile 'com.android.support:support-v4:19.1.0'
+    }
+
+    android {
+      compileSdkVersion 8
+      buildToolsVersion "19.0.3"
+
+      sourceSets {
+        main {
+          manifest.srcFile 'AndroidManifest.xml'
+          java.srcDirs = ['src']
+          res.srcDirs = ['res']
+        }
+      }
+    }
+}
