diff --git a/src/main/java/com/bourke/glimmrpro/common/Keys.java b/src/main/java/com/bourke/glimmrpro/common/Keys.java
index fb423b6..a59f93e 100644
--- a/app/src/main/java/com/bourke/glimmrpro/common/Keys.java
+++ b/app/src/main/java/com/bourke/glimmrpro/common/Keys.java
@@ -2,6 +2,6 @@ package com.bourke.glimmrpro.common;
 
 public class Keys {
 
-    public static final String API_KEY = "";
-    public static final String API_SECRET = "";
+    public static final String API_KEY = "92e129a9a1c2fa92089c841338dfa0b4";
+    public static final String API_SECRET = "81400eccae9d3c9b";
 }
