diff --git a/tesseract-android-tools/jni/Android.mk b/tesseract-android-tools/jni/Android.mk
index a38bc4f..c649ef0 100755
--- a/tesseract-android-tools/jni/Android.mk
+++ b/tesseract-android-tools/jni/Android.mk
@@ -13,9 +13,9 @@
 # LEPTONICA_PATH := <path-to-leptonica>
 # LIBJPEG_PATH := <path-to-libjpeg>
 
-#TESSERACT_PATH := $(call my-dir)/../external/tesseract-3.00
-#LEPTONICA_PATH := $(call my-dir)/../external/leptonlib-1.66
-#LIBJPEG_PATH := $(call my-dir)/../external/libjpeg
+TESSERACT_PATH := $(call my-dir)/../external/tesseract-3.01
+LEPTONICA_PATH := $(call my-dir)/../external/leptonlib-1.66
+LIBJPEG_PATH := $(call my-dir)/../external/libjpeg
 
 ifeq "$(TESSERACT_PATH)" ""
   $(error You must set the TESSERACT_PATH variable to the Tesseract source \
