diff --git a/app/src/dacer/settinghelper/SettingUtility.java b/app/src/dacer/settinghelper/SettingUtility.java
index 281bb2b..3aff17f 100644
--- a/app/src/dacer/settinghelper/SettingUtility.java
+++ b/app/src/dacer/settinghelper/SettingUtility.java
@@ -278,8 +278,7 @@ public class SettingUtility {
 	
 	
     public static boolean isLightTheme(){
-        boolean value = SettingHelper.getSharedPreferences(getContext(), THEME_TYPE, "black").equals("white");
-        return value;
+        return true;
     }
     
     public static boolean isVibrator() {
