diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index db71471..83a7e13 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -27,7 +27,7 @@
 
     <uses-feature
         android:name="android.hardware.nfc"
-        android:required="true" />
+        android:required="false" />
 
     <application
         android:allowBackup="false"
@@ -54,4 +54,4 @@
         </activity>
     </application>
 
-</manifest>
\ No newline at end of file
+</manifest>
diff --git a/res/values/strings.xml b/res/values/strings.xml
index 556360e..5eb1f9d 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -17,14 +17,14 @@
 <resources>
 
     <string name="app_name">Clipboard Beam</string>
-    <string name="touch_phones">Touch phones&#8230;</string>
-    <string name="activate_nfc">Please activate NFC</string>
+    <string name="touch_phones">Touch phones or enter some text&#8230;</string>
+    <string name="activate_nfc">Maybe you need to activate NFC?</string>
     <string name="menu_share">Share</string>
     <string name="text_saved">Text saved to clipboard</string>
     <string name="error">Error</string>
     <string name="info">Info</string>
-    <string name="info_long">%1$s reads your clipboard and beams the content to your buddy with Android Beam&#8482;.\n\nIt\'s free. It\'s open source. It\'s GPLv3 (%2$s).\n\nGet the source on github: %3$s</string>
+    <string name="info_long">%1$s reads your clipboard and beams the content to your buddy with Android Beam&#8482;.\n\nIt\'s free. It\'s open source. It\'s GPLv3 (%2$s).\n\nGet the source on github.com: %3$s</string>
     <string name="link_gpl3" translatable="false">http://www.gnu.org/licenses/gpl-3.0.txt</string>
-    <string name="link_github" translatable="false">https://gihub.com/felixb/clipboard-beam/</string>
+    <string name="link_github" translatable="false">https://github.com/felixb/clipboard-beam/</string>
 
-</resources>
\ No newline at end of file
+</resources>
diff --git a/src/de/ub0r/android/clipboardbeam/MainActivity.java b/src/de/ub0r/android/clipboardbeam/MainActivity.java
index 68c83c0..b9d36ce 100644
--- a/src/de/ub0r/android/clipboardbeam/MainActivity.java
+++ b/src/de/ub0r/android/clipboardbeam/MainActivity.java
@@ -63,55 +63,12 @@ public class MainActivity extends Activity implements CreateNdefMessageCallback
 		setContentView(R.layout.activity_main);
 
 		mEt = (EditText) findViewById(R.id.editText1);
-		if (savedInstanceState != null) {
-			mEt.setText(savedInstanceState.getString("text"));
-		} else {
-			CharSequence text = null;
-			ClipboardManager cb = (ClipboardManager) getSystemService(CLIPBOARD_SERVICE);
-			Intent i = getIntent();
-			String action = i.getAction();
-			if (action != null
-					&& action.equals(NfcAdapter.ACTION_NDEF_DISCOVERED)) {
-				// got NFC, save to mET & CB
-				text = null;
-				Parcelable[] rawMsgs = i
-						.getParcelableArrayExtra(NfcAdapter.EXTRA_NDEF_MESSAGES);
-				if (rawMsgs != null && rawMsgs.length > 0) {
-					NdefMessage msg = (NdefMessage) rawMsgs[0];
-					NdefRecord[] records = msg.getRecords();
-					if (records != null && records.length > 0) {
-						try {
-							text = parseNdef(records[0]);
-						} catch (UnsupportedEncodingException e) {
-							Log.e(TAG, "encoding error", e);
-							Toast.makeText(this, R.string.error,
-									Toast.LENGTH_LONG).show();
-							text = null;
-						}
-					}
-				}
-				if (text != null) {
-					text = text.toString().trim();
-				}
-				cb.setPrimaryClip(new ClipData("text",
-						new String[] { "text/plain" }, new Item(text)));
-				Toast.makeText(this, R.string.text_saved, Toast.LENGTH_LONG)
-						.show();
-			} else if (cb.hasPrimaryClip()) {
-				text = cb.getPrimaryClip().getItemAt(0).coerceToText(this);
-			}
-			if (text != null) {
-				mEt.setText(text.toString().trim());
-			}
-		}
 
 		mNfcAdapter = NfcAdapter.getDefaultAdapter(this);
 		if (mNfcAdapter == null) {
 			// point to NFC settings
 			Toast.makeText(this, R.string.activate_nfc, Toast.LENGTH_LONG)
 					.show();
-			startActivity(new Intent(
-					android.provider.Settings.ACTION_WIRELESS_SETTINGS));
 		} else {
 			mNfcAdapter.setNdefPushMessageCallback(this, this, this);
 		}
